network/controller/bootstrap/bootstrap.go

```diff
@@ -294,11 +299,23 @@ func (bc *Bootstrapper) startBootstrap(address string) (*host.Host, error) {
 		return nil, errors.Wrapf(err, "Failed to get response to bootstrap request from address %s", address)
 	}
 	data := response.GetData().(*NodeBootstrapResponse)
-	if data.Code == Rejected {
+	switch data.Code {
+	case Rejected:
 		return nil, errors.New("Rejected: " + data.RejectReason)
-	}
-	if data.Code == Redirected {
+	case Redirected:
 		return bootstrap(data.RedirectHost, bc.options, bc.startBootstrap)
+	case Accepted:
+		request := bc.transport.NewRequestBuilder().Type(types.FakePulsarRequest).Data(fakepulsar.Request{}).Build()
```

Comment:
With moved pulse data to bootstrap response you could remove that code.

Reply:
removed